src/main/kotlin/com/openlattice/linking/BackgroundLinkingService.kt

```diff
@@ -70,10 +72,10 @@ class BackgroundLinkingService(
     private val priorityEntitySets = configuration.whitelist.orElseGet { setOf() }
 
     @Suppress("UNUSED")
-    private val enqueuer = executor.submit {
-
-        try {
-            while (true) {
+    @Scheduled(fixedRate = LINKING_RATE)
+    fun enqueue() {
+        executor.submit {
```

Comment:
This schedules a scheduling was that intended?3

Reply:
I'm not sure what that means 🤔 
Is `executor.submit{` in `@Scheduled` a broken or otherwise bad pattern?

Reply:
Ah, I see. Wasn't sure how @Scheduled gets configured. Should be all good now